covalic/webroot.mako

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Covalic</title>
+    <link rel="stylesheet" href="/static/built/girder_lib.min.css">
+    <link rel="icon" type="image/png" href="/static/built/Girder_Favicon.png">
+    % for plugin in pluginCss:
+    <link rel="stylesheet" href="/static/built/plugins/${plugin}/plugin.min.css">
+    % endfor
+  </head>
+  <body>
+    <div id="g-global-info-apiroot" class="hide">/api/v1ZZ</div>
```

Comment:
ZZ?

Reply:
Typo, fixed.

Reply:
I guess our regression tests don't cover this stuff, have we verified the web client manually?

Reply:
No.

With the current state of devops, I can't spin up a Vagrant box and I can't deploy non-master branches to servers. We're currently working on improving the deployability, and as you see from #249, I've put a lot of effort into improving / fixing tests. I believe @danlamanna has some thoughts about continuous release, which we may be discussing later today in another venue.

However, for now, unless you strongly object, I'm going to merge this just so we can hopefully fix the one and only production server that's running master + Girder 3.

Reply:
Fine by me